### PR TITLE
[BZ 2048763]: PR to update - IBM Power Systems to IBM Power

### PR DESCRIPTION
**Pull request by alishaIBM**
- OCP version for cherry-picking: enterprise-4.9, 4.10

    Bugzilla : https://bugzilla.redhat.com/show_bug.cgi?id=2048763

    Preview : https://deploy-preview-41204--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html

    QE review : Luvella McFadden

### CODE REVIEW

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [id="installing-ibm-power"]
```
Comment (SNiemann15) [inline installing/installing_ibm_power/installing-ibm-power.adoc:1]: insert new line before id entry line:
```
:_content-type: ASSEMBLY
[id="installing-ibm-power"]
```

File: installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc [hunk 2]
```diff
@@ -1,12 +1,12 @@
 [id="installing-restricted-networks-ibm-power"]
```
Comment (SNiemann15) [inline installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc:1]: insert new line before id entry line:
```
:_content-type: ASSEMBLY
[id="installing-restricted-networks-ibm-power"]
```

**Comment by lsmcfadden** [comment]
@manojnkumar this looks fine to me but do you have any concern with 'readability'.  I think the guidance would allow us to say IBM Power system or IBM Power server if you have grave concerns.  I see **IBM Z** is also used w/o any trailing "Systems" and it reads fine to me.  I looked at it from this view (https://github.com/openshift/openshift-docs/pull/41204/files) so I could see how the sentences read with and w/o the trailing "Systems".

**Review by opayne1 — COMMENTED** [review]
LGTM!

**Pull request merged**